lib/components/viewers/viewers.css

```diff
@@ -259,7 +259,7 @@
   color: #111;
   display: flex !important;
   height: 50px;
-  margin-bottom: -5px !important;
+  margin-bottom: 0px !important;
```

Comment:
This is causing the clear button to become misaligned <img width="71" alt="Screenshot 2025-05-28 at 2 47 29 PM" src="https://github.com/user-attachments/assets/6511d135-255c-4f4d-8a59-a52cda09442d" />